client/settings/payment-methods-selector/style.scss

```diff
@@ -3,39 +3,13 @@
 		margin: 24px -24px;
 	}
 
-	&__list-item {
```

Comment:
Moved to `client/components/payment-methods-checkboxes/payment-method-checkbox.scss`